src/promised.js

```diff
@@ -63,17 +63,14 @@ function decodeString(str) {
   });
 }
 
-
 const writeFile = async resultTxt => {
   const select = parseInt(await question('Print 1 to save results\n'));
```

Comment:
You can easily create `questionInt` to hide `parseInt` inside it's implementation so don't need to wrap `await question` into `parseInt` each time